src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlTransaction.cs

```diff
@@ -2,13 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
 using System.ComponentModel;
 using System.Data;
 using System.Data.Common;
 using System.Diagnostics;
 using System.Runtime.CompilerServices;
 using Microsoft.Data.Common;
+using static Microsoft.Data.SqlClient.SqlClientEventSource;
```

Comment:
Can this using be avoided since we are in the same package?

Reply:
I am still not sure about this. I can take this out and add SqlClientEventSource before Log in the context.

Reply:
According to` https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-static ` The using static directive was introduced in C# 6. 

> using static imports only accessible static members and nested types declared in the specified type. Inherited members are not imported. You can import from any named type with a using static directive, including Visual Basic modules. If F# top-level functions appear in metadata as static members of a named type whose name is a valid C# identifier, then the F# functions can be imported.
> 
> using static makes extension methods declared in the specified type available for extension method lookup. However, the names of the extension methods are not imported into scope for unqualified reference in code.

Reply:
My question would be whether it's worth that extra static using directive at the top of the file just to avoid the longer form which is only `SqlClientEventSource.Log.Method` and I think the longer form is clearer because it's obvious that the Log object is linked to the EventSource system and not some sort of file log.

Reply:
Well, I believe Taking that out from the top and implement it inside the code  is a better approach. In cases that logging is not enabled it wont be called at all and it is always contained in the if statement.

Reply:
But still in some cases the namespace without static word should be declared.

Reply:
Let's do that then, I also think the longer form is better in that way, it rules out the need for static reference and also gives context to "Log". For classes outside package, we can continue to use `using Microsoft.Data.SqlClient` and keep the same format.

Reply:
Done!